virtcontainers/qemu_s390x.go

```diff
@@ -95,18 +96,28 @@ func (q *qemuS390x) bridges(number uint32) {
 	q.Bridges = genericBridges(number, q.machineType)
 }
 
-// appendBridges appends to devices the given bridges
-func (q *qemuS390x) appendBridges(devices []govmmQemu.Device) []govmmQemu.Device {
-	return genericAppendBridges(devices, q.Bridges, q.machineType)
-}
-
 // appendConsole appends a console to devices.
 // The function has been overwriten to correctly set the driver to the CCW device
 func (q *qemuS390x) appendConsole(devices []govmmQemu.Device, path string) []govmmQemu.Device {
+	id := "serial0"
+	addr, b, err := q.addDeviceToBridge(id, types.CCW)
+	if err != nil {
+		logrus.Errorf("Failed to append console: %v", err)
```

Comment:
I think you can use the `virtLog` like in `qemu_amd64.go`, also please use `WithError(err)`

Reply:
I will change it accordingly in the file